app/controllers/emergency_contacts_controller.rb

```diff
@@ -1,2 +1,39 @@
 class EmergencyContactsController < ApplicationController
+    def index
```

Comment:
The variable you define and the variable you render as json needs to be the same. I don't image you are referencing contacts outside the scope of this class so you likely don't need the instance variable